docs/Admin/Test-ExchangePropertyPermissions.md

```diff
@@ -0,0 +1,30 @@
+# Test-ExchangePropertyPermissions
+
+Download the latest release: [Update-Engines.ps1](https://github.com/microsoft/CSS-Exchange/releases/latest/download/Update-Engines.ps1)
+
+## Description
+
+Test-ExchangePropertyPermissions is designed to detect certain schema issues which can manifest as
+permissions problems and can be challenging to identify manually, including:
+
+* Scenarios where a property set does not include all the expected properties.
+* Scenarios where an objectClass definition is missing expected properties.
+
+Note that the script does not perform an exhaustive check for all possible schema issues. It is
+only designed to identify a specific subset of issues which we have encountered. For example, using
+AD Schema Analyzer as described here is one such scenario:
+
+https://learn.microsoft.com/en-us/previous-versions/technet-magazine/dd547839(v=msdn.10)
+
+As noted in that article, this is known to corrupt the Exchange attributes. This script is able
+to detect that scenario, and other similar scenarios.
+
+Further, note that such issues cannot be fixed by the script. Using AD Schema Analyzer as described
+results in an unsupported forest that should be torn down.
+
+## Example
+
+.\Test-ExchangePropertyPermissions.ps1 -TargetObjectDN "CN=SomeRecipient,OU=Users,DC=contoso,DC=com" -ComputerAccountDN "CN=SomeServerName,OU=Computers,DC=contoso,DC=com"
```

Comment:
Should probably also include a `Syntax` section. Similar to what we have for a few other scripts.

Reply:
Done.